src/redux/reducers/todosReducer.js

```diff
@@ -1,13 +1,31 @@
+import { ADD_TODO, IS_ADDING } from "redux/actions/actionTypes"
 export const initialState = {
-  todosItems: [
-    { text: "Eat", completed: true },
-    { text: "Sleep", completed: false },
-  ],
+  todoItems: [],
   visibilityFiler: "SHOW_ALL",
+  isAdding: false,
 }
 
 export default function todosReducer(state = initialState, action) {
   switch (action.type) {
+    case ADD_TODO:
+      const { text, createdAt, completed } = action.payload
```

Comment:
Why do we need to store destructured data into temporary variable ?

Reply:
I think storing it as variables first makes it easier to understand , also allows us to pass in named values as shown in the picture



![Screenshot 2023-08-27 at 2 37 49 PM](https://github.com/umh-wtag/todo-app/assets/139974355/45d1c609-887c-40e3-aeb7-c90843988464)